src/Tribe/Aggregator.php

```diff
@@ -582,4 +593,13 @@ public function hook() {
 
 		return true;
 	}
+
+	/**
+	 * Function used to remove cache stored in transients.
+	 *
+	 * @since TBD
+	 */
+	public function clear_services_list_cache() {
+		delete_transient( self::SERVICES_CACHE_KEY );
```

Comment:
`return` the result o the function call.

Reply:
👍